server/odc-service/src/main/java/com/oceanbase/odc/service/regulation/ruleset/SqlConsoleRuleService.java

```diff
@@ -128,7 +128,7 @@ public <T> Optional<List<T>> getListProperties(@NonNull Long rulesetId, @NonNull
                 return true;
             }).map(rule -> {
                 if (Objects.isNull(rule.getProperties())) {
-                    return Collections.EMPTY_LIST;
+                    return null;
```

Comment:
why null, does empty list not work?

Reply:
we need to distinguish 'null' and 'empty list'. 'Null' means that this rule is disabled while 'empty list' means that this rule is enabled but the value is empty.

Reply:
okay